cGame_logic.cpp

```diff
@@ -17,6 +17,13 @@
 
 #include <fmt/core.h>
 
+namespace {
+
+constexpr auto kMinAlpha = 0;
```

Comment:
why the prefix `k`?  (just curious)

Reply:
Oh, that's my work coding guidelines creeping in. We use that to signify it's a constant that is the same for the whole program duration. I believe it comes from the Google C++ Style Guide. See: https://google.github.io/styleguide/cppguide.html#Constant_Names

Now, I don't think the Google Style Guide is the be all and end all of style guides, but this one seems useful to me.